examples/structured_2d_dgsem/elixir_advection_coupled.jl

```diff
@@ -1,5 +1,6 @@
 using OrdinaryDiffEq
 using Trixi
+import Trixi.Indexing
```

Comment:
If the coupling stuff becomes non-experimental, we should export this from Trixi.jl.

```suggestion
import Trixi.Indexing # coupling is an experimental feature
```

Reply:
What not
```suggestion
using Trixi: Indexing
```
Not a big fan of `import` and it seems like the Julia community is slowly eschewing `import`. But it's not a big deal either IMHO, so we can also just leave it as it is.